lib/stripe/stripe_object.rb

```diff
@@ -43,7 +43,7 @@ def inspect
 
     def refresh_from(values, opts, partial=false)
       @opts = opts
-      @previous_metadata = values[:metadata]
+      @previous = values
```

Comment:
Re: deep copy, doesn't convert_to_stripe_object already deep-copy the values for us, so this is the only reference left to the original data?